src/devices/CharacterLcd/samples/Pcf8574tSample.cs

```diff
@@ -0,0 +1,353 @@
+﻿using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+using System.Timers;
+using Iot.Device.Mcp23xxx;
+using Iot.Device.CharacterLcd;
+
+namespace Iot.Device.CharacterLcd.Samples
+{
+    class Pcf8574tSample

```

Comment:
Can we remove the nesting of classes? I would only keep one class called Pcf8574tSample with the main method inside.